pkg/minikube/node/config.go

```diff
@@ -79,3 +77,34 @@ func configureMounts(wg *sync.WaitGroup, mount bool, mountString string) {
 		exit.Error(reason.HostMountPid, "Error writing mount pid", err)
 	}
 }
+
+func generateMountArgs(profile string, cc config.ClusterConfig) []string {
+	mountDebugVal := 0
+	if klog.V(8).Enabled() {
+		mountDebugVal = 1
+	}
+
+	args := []string{"mount", cc.MountString}
+	flags := []struct {
+		name  string
+		value string
+	}{
+		{"profile", profile},
+		{"v", fmt.Sprintf("%d", mountDebugVal)},
+		{constants.Mount9PVersionFlag, cc.Mount9PVersion},
+		{constants.MountGIDFlag, cc.MountGID},
+		{constants.MountIPFlag, cc.MountIP},
+		{constants.MountMSizeFlag, fmt.Sprintf("%d", cc.MountMSize)},
+		{constants.MountModeFlag, fmt.Sprintf("%d", cc.MountMode)},
+		{constants.MountPortFlag, fmt.Sprintf("%d", cc.MountPort)},
+		{constants.MountTypeFlag, cc.MountType},
+		{constants.MountUIDFlag, cc.MountUID},
+	}
+	for _, flag := range flags {
```

Comment:
are there any weird side effects if only some of the mount options are set and not others? what happens if one of these is an empty string?

Reply:
Good question, but the command runner seems to handle empty strings fine, `MountIP` is empty by default and the mounting works fine.